functions/New-DbaDbDataGeneratorConfig.ps1

```diff
@@ -136,7 +136,7 @@ function New-DbaDbDataGeneratorConfig {
 
             # Get the tables
             if ($Table) {
-                $tablecollection = $db.Tables | Where-Object Name -in $Table
+                $tablecollection = $db | Get-DbaDbTable -Table $Table
```

Comment:
I'm not sure if this makes the table collection any faster. The collection is already there, why would you run another command that would do the same thing?

Reply:
Good shout. I agree